dht_test.go

```diff
@@ -1158,101 +1160,91 @@ func TestClientModeFindPeer(t *testing.T) {
 	}
 }
 
+func minInt(a, b int) int {
+	if a < b {
+		return a
+	} else {
+		return b
+	}
+}
+
+func TestFindPeerQueryMinimal(t *testing.T) {
+	testFindPeerQuery(t, 2, 22, 11)
+}
+
 func TestFindPeerQuery(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+	if curFileLimit() < 1024 {
+		t.Skip("insufficient file descriptors available")
```

Comment:
Why was this necessary?

Reply:
I wanted to leave the stress test in for now, but keep it only on platforms that can cope.